src/sif/motorscootercost.cc

```diff
@@ -426,9 +426,11 @@ Cost MotorScooterCost::EdgeCost(const baldr::DirectedEdge* edge,
     return {sec * ferry_factor_, sec};
   }
 
+  // prevent scooter speed to become 0
   uint32_t scooter_speed =
-      (std::min(top_speed_, speed) * kSurfaceSpeedFactors[static_cast<uint32_t>(edge->surface())] *
-       kGradeBasedSpeedFactor[static_cast<uint32_t>(edge->weighted_grade())]);
+      std::max(1.f, (std::min(top_speed_, speed) *
```

Comment:
at least 1. alternatively we could leave this std::max and update the `speedfactor_[0]` to the same value as `speedfactor_[1]` (3.6), which is currently set to 3600.